dymos/transcriptions/pseudospectral/radau_pseudospectral.py

```diff
@@ -260,10 +271,24 @@ def configure_timeseries_outputs(self, phase):
                               tgt_name=f'{timeseries_name}.input_values:states:{state_name}',
                               src_indices=om.slicer[src_rows, ...])
 
-                rate_src, src_idxs = self.get_rate_source_path(state_name, 'all', phase)
-                phase.connect(src_name=rate_src,
-                              tgt_name=f'{timeseries_name}.input_values:state_rates:{state_name}',
-                              src_indices=src_idxs)
+                rate_src = options['rate_source']
+                if rate_src in phase.parameter_options:
+                    # If the rate source is a parameter, which is an input, we need to promote
+                    # the state rates input value to the parameter name instead of connecting to it.
+                    nn = self.grid_data.subset_num_nodes['all']
+                    shape = phase.parameter_options[rate_src]['shape']
```

Comment:
Just refreshing my memory, but at this point in the configure stack, the `shape` stored in parameter_options has been interrogated from the model and stored here?

Reply:
That's the intent.  States are interrogated last in case they depend on any other variables.